libs/providers/launchdarkly-client/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@openfeature/launchdarkly-client-provider",
+  "version": "0.0.1",
+  "type": "commonjs",
+  "scripts": {
+    "publish-if-not-exists": "cp $NPM_CONFIG_USERCONFIG .npmrc && if [ \"$(npm show $npm_package_name@$npm_package_version version)\" = \"$(npm run current-version -s)\" ]; then echo 'already published, skipping'; else npm publish --access public; fi",
+    "current-version": "echo $npm_package_version"
+  },
+  "peerDependencies": {
+    "@openfeature/web-sdk": "^0.3.2-experimental",
+    "launchdarkly-js-client-sdk": "^3.1.3"
+  },
+  "dependencies": {
+    "lodash.isequal": "^4.5.0"
+  },
+  "devDependencies": {
+    "@types/lodash.isequal": "^4.5.6"
+  }
```

Comment:
```suggestion
  }
```

Reply:
These should compute automatically if you add them to the root package.json

Reply:
Done
I did not think this was going to work, is not even in the peer, how ?

Reply:
nx _should_ automatically compute these and put them in the distributed modules package.json. That is how it worked until a recent bug in nx. Upgrading to nx v16 fixes it. I'll do that in a separate PR and make sure everything works again before releasing. I've rebased you branch on main and resolved some conflicts. I'm OK to merge now unless you have any objections.

Reply:
@Mateoc FYI, this is how things SHOULD work: https://github.com/open-feature/js-sdk-contrib/pull/362

It generates the following deps for you: 
![image](https://github.com/open-feature/js-sdk-contrib/assets/25272906/c00b7dbc-e03a-42dc-963e-fb4c343b6341)